extensions/amp-document-recommendations/0.1/amp-document-recommendations.js

```diff
@@ -164,6 +164,8 @@ export class AmpDocumentRecommendations extends AMP.BaseElement {
     return this.getVsync()
         .mutatePromise(() => {
           const shadowRoot = this.win.document.createElement('div');
+          const marker = doc.createElement(MARKER_TAG);
+          doc.body.appendChild(marker);
```

Comment:
Shouldn't you be attaching this right after the component? Otherwise when we remove the placement restrictions this will stop working.

Reply:
I'm using it to mark the end of the document, rather than where the document-recommendations tag is placed. Later on we'll want to put it where the break is going, but that won't be useful until we're hiding everything below it.